src/ui/pages/work/Workorder.js

```diff
@@ -343,6 +343,8 @@ class Workorder extends Entity {
                         handleError={handleError}
                         userCode={userData.eamAccount.userCode}
                         disabled={this.departmentalSecurity.readOnly}
+                        hideFilledItems={panelQueryParams.hideFilledItems}
```

Comment:
The component should receive a boolean for this prop. Do the required conversion before if necessary